ts/Grid/Core/Options.ts

```diff
@@ -222,9 +222,17 @@ export interface RowsSettings {
      * only. In case of large data set, the enabled option improve performance
      * and saves memory.
      *
-     * @default true
+     * @default false
      */
     virtualization?: boolean;
+
+    /**
+     * The rows virtualization threshold option. Based on the row count,
```

Comment:
Maybe something more like this?
```
The rows virtualization threshold option sets the row count limit at which virtualization is activated.
When the number of rows exceeds this threshold, virtualization is enabled to optimize performance.
```

Also please describe in both which takes precedence over which.